### ISSUE TITLE
Walks the wrong way after the sandwich

### ISSUE THREAD

**Issue by blackcatsnow** [mwoerner77/SVEggBot#2]
Occasionally the bot will walk towards the basket but will either not quite make it or miss it entirely. If I use the stick to move it to the right place it will work as expected.

**Comment by blackcatsnow** [comment]
I also realized that you have to make sure to press ZL before starting the bot or it will walk to a random direction and not straight because the bot commands the system to move the left stick up. I am also having the issue of not making a new sandwich after 30 minutes. Not sure if this is intended or not as the previous issue mentioned.

**Comment by mwoerner77** [comment]
There appears to be a discrepancy with delays based on different switches and setups. I need to update the app to normalize movement to the basket with ZL camera resets and make delays configurable by the user.

**Comment by blackcatsnow** [comment]
My apologies. I meant just L. I wrote the wrong button. But yes I agree! Thanks for replying!

**Comment by mwoerner77** [comment]
I have updated the bot. Are you able to fix the issue now that delays are configurable?

**Comment by mwoerner77** [comment]
Problem should be fixed in version 1.1.0. Open a new issue if you're still having this problem.
